### ISSUE TITLE
Widen Month select box to fit full month name width

### ISSUE THREAD

**Issue by CrystabelReiter** [department-of-veterans-affairs/va.gov-team#20546]
Widen the length of the month select box in the date input component for month/day/year and month/year from 10 rem to 12 rem 

Example ticket: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/588

**Comment by caw310** [comment]
@bkjohnson , what is the status of this? Is this in storybook and the forms library??

**Comment by k80bowman** [comment]
All of the month select boxes I could find use the formation class `.form-datefield-month`, which was updated to 12rem in the PR linked above. So all month select boxes were already updated to 12rem, in both the design system and the forms system.

However, I found that 12rem was not quite enough. The last few pixels of the longer month names were still being cut off, as seen here:
![image.png](https://images.zenhubusercontent.com/5952ac1861b4ca3b0a727ca8/ec1b5bf3-1df5-4b03-84a0-6dbb7da8bfce)

I've submitted a [PR to increase the width to 13rem](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/608), which is enough to display the full month name
![image.png](https://images.zenhubusercontent.com/5952ac1861b4ca3b0a727ca8/931622b6-e1ed-4efc-92ed-2a24859c5fb5)
